packages/menu/src/spectrum-config.js

```diff
@@ -62,10 +79,38 @@ const config = {
                 converter.classToAttribute('is-focused', 'focused'),
                 converter.classToAttribute('is-selected', 'selected'),
                 converter.classToId('spectrum-Menu-itemLabel', 'label'),
-                converter.classToClass('spectrum-Menu-itemIcon', 'icon'),
                 converter.classToClass('spectrum-Menu-chevron', 'chevron'),
+                converter.classToClass(
+                    'spectrum-Menu-chevron--withAdjacentIcon',
+                    'chevron--withAdjacentIcon'
+                ),
                 converter.classToClass('spectrum-Menu-checkmark', 'checkmark'),
+                converter.classToClass(
+                    'spectrum-Menu-checkmark--withAdjacentIcon',
+                    'checkmark--withAdjacentIcon'
+                ),
+                converter.classToAttribute(
+                    'spectrum-Menu-item--drillIn',
+                    'has-submenu'
+                ),
```

Comment:
If they call these `drillIn` and we call them `submenu`, will there be any naming issues when we move to support the "expanded menu"?

Reply:
I was wondering that at first, as well, but I made the decision not to bring it up with the team 🤔 This is because it wouldn't affect any of the other work that needs to happen down the line since it's really just for the submenu and that's it. The other work that needs to happen is from the collapsible stuff, which uses a different class name, and the descriptive text below the main menuitem label.

Reply:
Sounds good!